src/view/components/Drawer/DrawerHeader/index.tsx

```diff
@@ -1,32 +1,35 @@
 import * as React from 'react';
```

Comment:
sorry i missed this file when i cleaned the files i updated :) thanks for covering this

Reply:
Tbh I cleaned the whole file without looking at the name, just because it was in header haha